Tests/RevenueCatUITests/CustomerCenter/CustomerCenterViewModelTests.swift

```diff
@@ -128,6 +128,56 @@ class CustomerCenterViewModelTests: TestCase {
         }
     }
 
+    func testAppIsLatestVersion() {
+        let testCases = [
+            (currentVersion: "1.0.0", latestVersion: "2.0.0", expectedAppIsLatestVersion: false),
+            (currentVersion: "2.0.0", latestVersion: "2.0.0", expectedAppIsLatestVersion: true),
+            (currentVersion: "3.0.0", latestVersion: "2.0.0", expectedAppIsLatestVersion: true),
+            (currentVersion: "1.0.0", latestVersion: "1.1.0", expectedAppIsLatestVersion: false),
+            (currentVersion: "1.1.0", latestVersion: "1.1.0", expectedAppIsLatestVersion: true),
+            (currentVersion: "1.1.0", latestVersion: "1.0.0", expectedAppIsLatestVersion: true),
+            (currentVersion: "1.0.0", latestVersion: "1.0.1", expectedAppIsLatestVersion: false),
+            (currentVersion: "1.0.1", latestVersion: "1.0.1", expectedAppIsLatestVersion: true),
+            (currentVersion: "1.0.1", latestVersion: "1.0.0", expectedAppIsLatestVersion: true),
+            // The CFBundleVersion docs state:
+            // > You can include more integers but the system ignores them.
+            // https://developer.apple.com/documentation/bundleresources/information_property_list/cfbundleversion
+            // So we should do the same.
+            (currentVersion: "2.0.0.2.3.4", latestVersion: "2.0.0.3.4.5", expectedAppIsLatestVersion: true),
+            (currentVersion: "1.0.0.2.3.4", latestVersion: "2.0.0.3.4.5", expectedAppIsLatestVersion: false),
+            (currentVersion: "1.2", latestVersion: "2", expectedAppIsLatestVersion: false),
+            (currentVersion: "1.2", latestVersion: "1", expectedAppIsLatestVersion: true),
+            (currentVersion: "2", latestVersion: "1", expectedAppIsLatestVersion: true),
+            (currentVersion: "0", latestVersion: "1", expectedAppIsLatestVersion: false),
+            (currentVersion: "10.2", latestVersion: "2", expectedAppIsLatestVersion: true),
+            // We default to true if we fail to parse any of the two versions.
+            (currentVersion: "not-a-number", latestVersion: "not-a-number-either", expectedAppIsLatestVersion: true),
+            (currentVersion: "not-a-number", latestVersion: "1.2.3", expectedAppIsLatestVersion: true),
+            (currentVersion: "1.2.3", latestVersion: "not-a-number", expectedAppIsLatestVersion: true),
+            (currentVersion: nil, latestVersion: nil, expectedAppIsLatestVersion: true),
+            (currentVersion: "1.2.3", latestVersion: nil, expectedAppIsLatestVersion: true),
+            (currentVersion: nil, latestVersion: "1.2.3", expectedAppIsLatestVersion: true)
+        ]
+        for (currentVersion, latestVersion, expectedAppIsLatestVersion) in testCases {
```

Comment:
Can't wait until we can use parameterized tests from Swift Testing
https://github.com/swiftlang/swift-testing?tab=readme-ov-file#scalable-coverage-and-execution

Reply:
Yea somehow I often default to parameterized tests. Easy to cover many bases. 

Swift Testing is Swift 6 only? We probably can't use that right away after it's released, right?